lib/bot.js

```diff
@@ -156,6 +156,9 @@ module.exports = (function() {
 				password = this.options.password;
 			}
 
+			// setup the domain
+			var domain = this.options.domain;
```

Comment:
Please update [README.md](https://github.com/macbre/nodemw#config-file) with the description of this option variable and please default it to an empty string here.